src/reason_pprint_ast.re

```diff
@@ -0,0 +1,6694 @@
+/*
```

Comment:
If you put this as the comment block of code at the top:
/*
 * Syntax and fallback syntax.
 * vim: set ft=rust:
 * vim: set ft=reason:
 */

Then it will highlight correctly.